FileShareService.DesktopClient.Core/Jobs/NewBatchJob.cs

```diff
@@ -52,11 +53,59 @@ public void Validate(JToken jsonToken)
             }
 
             //Check for files
-            if (jsonToken.SelectToken("actionParams.files") != null && 
+            if (jsonToken.SelectToken("actionParams.files") != null &&
                 jsonToken.SelectToken("actionParams.files").Type != JTokenType.Array)
             {
                 ErrorMessages.Add($"Invalid file object.");
             }
+
+            //Check for file attributes
+
+            foreach (JToken fileObj in jsonToken.SelectToken("actionParams.files"))
+            {
+                JToken? fileAttributeToken = fileObj.SelectToken("attributes");
+                var searchPath = fileObj.SelectToken("searchPath");
+
+                if (fileAttributeToken != null && JArray.FromObject(fileAttributeToken).Count > 0)
```

Comment:
can we invert this statement to reduce indentation? 

if (fileAttributeToken == null || JArray.FromObject(fileAttributeToken).Count == 0) continue;

Reply:
Changes suggested for indentation and attributes : null check are implemented